docs/application/native/guides/alarm/alarms.md

```diff
@@ -21,7 +21,7 @@ The main features of the Alarm API include:
 
 - Managing existing alarms
 
-  You can [list all scheduled alarms and cancel them](#scenario_4).
+  You can [list all scheduled alarms and also update or cancel them](#scenario_4).
```

Comment:
@kilig Kindly put a comma after 'alarms' in this sentence. Please write as "...scheduled alarms, and also update...."

Reply:
Done

Reply:
How about removing "also"? The result would be :

You can **list all scheduled alarms, and update or cancel them**.

Reply:
I think it is better

Reply:
Hello Ms. @mijin85cho, using 'also' will be better to emphasis, for clarity. However meaning is still the same of your suggested rephrase, we can surely incorporate it. Please let us know which one to incorporate here.

Reply:
Hello, @rishu-samsung 
I suggested to remove 'also' because we don't need to emphasis words here, as there are only 2 meaning group : "list all scheduled alarms" and "update or cancle them"
With only 2 items, to me, adding 'also' feels like emphasizing every point of the sentence so the sentence became heavy.
Without the 'also' we get same meaning as you said, with lighter sentence. 
So I suggested to remove it.

Reply:
Hello Ms. @mijin85cho, agreed with your point. Let's remove 'also' from this sentence.

@kilig Kindly incorporate the rephrase suggested by Ms. @mijin85cho.

Reply:
Done